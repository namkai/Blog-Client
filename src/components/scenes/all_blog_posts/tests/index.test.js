import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import { BlogPosts } from '../index';

it('renders correctly', () => {
	const blogposts = shallow(
		<BlogPosts />
	);
	expect(toJson(blogposts)).toMatchSnapshot();
});