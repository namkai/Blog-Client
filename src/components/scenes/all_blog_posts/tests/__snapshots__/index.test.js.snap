// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<BlogPostLayout
  card={<Component />}
>
  <Component
    query=""
    updateQuery={[Function]}
  />
  <Loading />
</BlogPostLayout>
`;
